### PR TITLE
Ds 78 finish desktop styling

### LINKED ISSUE #78: Finish Desktop Styling of entire pokeapp
**Is your feature request related to a problem? Please describe.**
Take care of any finishing touches to style of pokeapp, for deployment. e.g. spacing issues, cursor hovers, alignment of elements, colors, font sizes, etc.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Finish css styling for header and header elements
2. finish css styling for pokecard
3. finish css styling for modal and modal tabs
4. implement scroll bar logic for eevee in modal evolution tab

## Purpose
To have desktop styling look like high res figma prototype

Closes #78

### CODE REVIEW

File: src/components/TabBar.js [hunk 1]
```diff
@@ -35,15 +36,19 @@ export default function BasicTabs({
...
 
   return (
-    <Box sx={{ width: "100%" }}>
-      <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+    <Box component="div" sx={{ width: "100%" }}>
+      <Box>
         <Tabs
+          TabIndicatorProps={{
+            style: { background: pokemonColor }
+          }}
```
Comment (Gjhnsn) [inline src/components/TabBar.js:44]: indicator color looks good!

**Review by Gjhnsn — APPROVED** [review]
Updated styles are looking great!

File: src/components/EvolutionTab.js [hunk 2]
```diff
@@ -128,7 +130,21 @@ const EvolutionTab = ({ evolutionChainUrl }) => {
...
 
-  return <div className="spriteContainer">{spritesWithEvolutionArrows()}</div>;
+  return (
+    <>
+      <div
+        style={{
+          // custom scrollbar (firefox only)
+          scrollbarColor: `${pokemonColor} rgba(235, 232, 232, 0.4)`,
+          overflowX: "auto"
+          // ** unknown solution for safari and chrome **
```
Comment (Lap343) [inline src/components/EvolutionTab.js:140]: stinks that it doesn't work on there

File: src/components/Moves.js [hunk 3]
```diff
@@ -103,13 +103,20 @@ const Moves = ({ dexNo }) => {
     //get movetype icon
     const typeIcon = imageUrls[`${typeName}`];
 
+    // process move name to camel case - Capitalize each word
+    const camelize = (str) => {
+      return str.replace(/([A-Z]|\b\w)/g, function (word, index) {
```
Comment (Lap343) [inline src/components/Moves.js:108]: regex?

**Review by Lap343 — APPROVED** [review]
Nice using Regex to lowercase everything. CSS looks good too

File: src/components/EvolutionTab.js (hunk 2 above)
Comment (dannysanchez559) [reply src/components/EvolutionTab.js:140]: I lucked out since i like to use firefox as my main browser 👍

File: src/components/Moves.js (hunk 3 above)
Comment (dannysanchez559) [reply src/components/Moves.js:108]: yus

**Pull request merged**